backend/app/controllers/homes_controller.rb

```diff
@@ -22,26 +22,42 @@ def edit; end
   # POST /homes or /homes.json
   def create
     @home = Home.new(home_params)
+    # By definition, a new home's address has not been canonicalized.
+    @home.canonicalized = false
 
     respond_to do |format|
       if @home.save
         format.json { render :show, status: :created, location: @home }
+        CanonicalizeAddressJob.perform_later @home.id
```

Comment:
Conventionally this sort of thing is usually added as an [ActiveRecord callback](http://guides.rubyonrails.org/active_record_callbacks.html) in the relevant model.

Reply:
I think I updated this to do things with ActiveRecord callbacks – looks like it's behaving the way we want. Actually ends up being a bit neater which is nice